modules/jaeger-upgrading-es.adoc

```diff
@@ -0,0 +1,58 @@
+////
+This PROCEDURE module included in the following assemblies:
+- jaeger_install/rhbjaeger-updating
+- service_mesh_install/updating-ossm
+////
+
+[id="upgrading_es5_es6_{context}"]
+= Upgrading from Elasticsearch 5 to Elasticsearch 6
+
+There is an issue with certificates when upgrading from Elasticsearch 5 to Elasticsearch 6.
+
+As a workaround, you must delete your Jaeger instance, then recreate the Jaeger instance.  Re-creating the Jaeger instance triggers creating a new set of certificates.   If you are using persistent storage the same volumes can be mounted for the new Jaeger instance as long as the Jaeger name and namespace for the new Jaeger instance are the same as the deleted Jaeger instance.
+
+.Procedure if Jaeger is installed as part of Red Hat Service Mesh
+
+. Determine the name of your Jaeger custom resource file:
++
+ $ oc get jaeger -n istio-system
++
+You should see something like the following:
++
+  NAME     AGE
+  jaeger   3d21h
++
+. Copy the generated custom resource file into a temporary directory:
++
+  $ oc get jaeger jaeger -oyaml -n istio-system > /tmp/jaeger-cr.yaml
++
+. Delete the Jaeger instance:
++
+  $ oc delete jaeger jaeger -n istio-system
++
+. Recreate the Jaeger instance from your copy of the custom resource file:
++
+  $ oc create -f /tmp/jaeger-cr.yaml -n istio-system
```

Comment:
@JStickler we may request, to clean up the generated custom resource file at the end of execution.
```
$ rm /tmp/jaeger-cr.yaml
```